src/control.py

```diff
@@ -9,6 +9,11 @@
 # 4.2. Create the thread module, for example: NAME_OF_THREAD = Thread(target = VARIABLE_NAME.FUNCTION_NAME) 
 # Notice that the FUNCTION_NAME should be the function in the class (defined before), and is the one that you would like it to iterate
 # 4.3. Start the thread, for example: NAME_OF_THREAD.start()
+#-----------------------------------------------CODE COMPLETE---------------------------------------------#
+# class ph_control(object):
```

Comment:
Isn't ph_control() already defined as a class? Correct me if I'm wrong, but don't we need to instantiate an object of this class similar to you do below. I'm guessing an example might look like this: 
`import src.control as control
pH_controller = control.ph_control()
pH_thread = Thread(target = pH_controller.test())
pH_thread.start()
`
I like that you've made your example more general than mine, I just wanted to try and show the most important use case. 

Do threads only work for single functions?

Reply:
Here the aim of define ph_control(object) should just be an example for the users indicate that they should create a class. I will edit and try to make my sample code more obvious. 
For your last question, I think yes, I test before and the threads can only work for one single function within one class